statistics-service/build.gradle

```diff
@@ -0,0 +1,34 @@
+plugins {
+    id 'java'
+    id 'org.springframework.boot' version '3.0.5'
+    id 'io.spring.dependency-management' version '1.1.0'
+}
+
+group = 'com.adavydenko'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '17'
+
+configurations {
+    compileOnly {
+        extendsFrom annotationProcessor
+    }
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation project(':common')
```

Comment:
in multi module gradle project you may have a root build.gradle which will hold all common dependencies for all sub-modules.


```
-./build.gradle
-./settings.gradle
-./battle-service
   |--/build.gradle
-./user-service
   |--/build.gradle

```

so in root build.gradle you may put common dependencies, ie lombok,spring,springboot etc.